testsuite/ui/webdriver.py

```diff
@@ -120,9 +120,11 @@ def _get_selenium_driver(self):
                                               chrome_options=options)
         elif self.driver == 'firefox':
             browser_profile = webdriver.FirefoxProfile()
+            # default Firefox profile preference `open_newwindow` is set to 2. This causes the links open
+            # in the new window instead of a new tab.
+            browser_profile.DEFAULT_PREFERENCES["frozen"]["browser.link.open_newwindow"] = 3
```

Comment:
Something like this wouldnt work? `profile.set_preference("browser.link.open_newwindow", 3)`